.pre-commit-config.yaml

```diff
@@ -9,29 +9,34 @@ repos:
     -   id: check-yaml
         exclude: meta.yaml
     -   id: end-of-file-fixer
-        exclude: '(.*\.svg)'
+        exclude: '\.svg'
     -   id: mixed-line-ending
     -   id: trailing-whitespace
 -   repo: https://github.com/psf/black
     rev: stable
     hooks:
     -   id: black
-        files: '(\.pyi?|wscript)$'
+        files: '(\.pyi?|wscript|ipynb)$'
         language_version: python3.7
+        additional_dependencies: [black-nb]
 -   repo: https://github.com/asottile/blacken-docs
     rev: v1.7.0
     hooks:
     -   id: blacken-docs
         additional_dependencies: [black]
-        files: '(\.md|\.rst)$'
+        files: '(\.md|rst)$'
 -   repo: https://gitlab.com/pycqa/flake8
     rev: 3.8.3
     hooks:
     -   id: flake8
-        files: '(\.pyi?|wscript)$'
+        files: '(\.pyi?|wscript|ipynb)$'
+        exclude: (configurations|submodules|conf\.py|fig-eoq-sensitivity-analysis-example\.py)
         args: [
           '--max-line-length=100',
+          '--ignore=E203,E402,E501,E800,W503', # Black compatibility
+          '--select=B,C,E,F,W,T4,B9'
```

Comment:
These are copied from [econ-project-templates/.pre-commit-config.yaml](https://github.com/OpenSourceEconomics/econ-project-templates/blob/master/.pre-commit-config.yaml), to make flake8 and black work better together. Also see reference: [Black compatible configurations — Black 20.8b1 documentation](https://black.readthedocs.io/en/stable/compatible_configs.html?highlight=flake8#flake8)